plugins/de.cau.cs.kieler.klighd/plugin.xml

```diff
@@ -77,5 +77,12 @@
       </provider>
 
    </extension>
+   <extension
```

Comment:
Do we still need extension points @NiklasRentzCAU ?

Reply:
Yes. If something will be written to this setup (for example actions, syntheses etc. like in semantics) it will only be executed in Eclipse when used as an extension point. That was the main reason why we have this single programmatic extension mechanism now, that needs to be registered once as a Java Service and Eclipse extension point